docs/guides/governance/governance-model/GOVERNANCE-TEMPLATE-LARGE-TEAMS.md

```diff
@@ -34,23 +39,23 @@ Subset of contributors who have demonstrated a strong familiarity with the proje
 
 [List of current Triagers]([INSERT LINK TO GITHUB, GITLAB, ETC. TEAM ASSOCIATED WITH TRIAGERS])
  
-### Collaborator
+### Committer
 
-Subset of contributors who have been given write access to one or more project repositories. Both contributors and collaborators can propose changes to the project via pull requests, but only collaborators can formally review and approve (merge) these requests. Any contributor who has made a non-trivial contribution should be on-boarded as a collaborator in a timely manner. 
+Subset of contributors who have been given write access to one or more project repositories. Both contributors and committer can propose changes to the project via pull requests, but only committers can formally review and approve (merge) these requests. Any contributor who has made a non-trivial contribution should be on-boarded as a committer in a timely manner. 

```

Comment:
A comment that I made internally (see Slack thread) about the following:
> Any contributor who has made a non-trivial contribution should be on-boarded as a committer in a timely manner.

Some open-source projects might be very touchy about allowing write permissions to external contributors. Branch protection and permissions can be set in place to alleviate some concerns, but human error is always possible. For example, one could be concerned about this allowing external contributors to execute arbitrary code on self-hosted GitHub Action runners.

@riverma suggested this new phrasing, which I personally really like:
> Any contributor who has made a non-trivial contribution should be considered as a candidate for the committer role by the steering committee or the product manager.

Reply:
Thank you for your review @thomas-bc and your suggestions! Will incorporate the new phrasing 👍